tasks/update-code/unarchive.yml

```diff
@@ -4,6 +4,16 @@
     copy: no
     src: "{{ ansistrano_archived_file }}"
     dest: "{{ ansistrano_release_path.stdout }}"
+  when: ansistrano_unarchive_owner == "default" and ansistrano_unarchive_group == "default"
+
+- name: ANSISTRANO | Unarchive | Unarchive source with owner and group
+  unarchive:
+    copy: no
+    src: "{{ ansistrano_archived_file }}"
+    dest: "{{ ansistrano_release_path.stdout }}"
+    owner: "{{ ansistrano_unarchive_owner }}"
+    group: "{{ ansistrano_unarchive_group }}"
+  when: ansistrano_unarchive_owner != "default" and ansistrano_unarchive_group != "default"
```

Comment:
I think this should be an or, right?

Reply:
@ricardclau the reason for AND is because if they only define one of them, then the other will be set to the value of DEFAULT, which would result in the group or owner being incorrect right?
ex: `ansistrano_unarchive_owner = root` then `ansistrano_unarchive_group = DEFAULT`

Reply:
yeah but no unarchives would be executed in that case, better make it crash than silently carry on

Reply:
or... it perhaps might be better to add an explicit fail statement if one variable is set and not the other or something similar. What are your thoughts on that?

Reply:
thats sounds good @ricardclau I'll update it